### PR TITLE
feat(edgeless): add pressure support for brush mode

### PR DESCRIPTION
**Pull request by L-Sun**
https://github.com/toeverything/blocksuite/assets/20479050/090727e4-7cba-4f47-8d8f-3feed5e30caf

### CODE REVIEW

File: packages/blocks/src/surface-block/element-model/brush.ts [hunk 1]
```diff
@@ -56,6 +58,9 @@ export class BrushElementModel extends ElementModel<BrushProps> {
   @yfield()
   points: number[][] = [];
 
+  @yfield()
+  pressures: number[] = [];
```
Comment (doodlewind) [inline packages/blocks/src/surface-block/element-model/brush.ts:62]: For existing data without this field, we need to make sure it doesn't break - you can edit in try-blocksuite.vercel.app and export the snapshot to test it.

Just to double check with @doouding that I think this yfield is supposed to store using plain array in yjs, not YArray?

Comment (doouding) [reply packages/blocks/src/surface-block/element-model/brush.ts:62]: > For existing data without this field, we need to make sure it doesn't break - you can edit in try-blocksuite.vercel.app and export the snapshot to test it.
> 
> Just to double check with @doouding that I think this yfield is supposed to store using plain array in yjs, not YArray?

Yes， this is going to be plain array. 

I wonder if the pressure could be store in the points array, make it a [number, number, number] array.

Comment (L-Sun) [reply packages/blocks/src/surface-block/element-model/brush.ts:62]: It is okay to store it in a points array, but I don't know whether this is a breaking change. I will test this later.

Comment (L-Sun) [reply packages/blocks/src/surface-block/element-model/brush.ts]: Currently, pressure and coordinates are stored in the `points` array, and it is compatible with previous snapshots.

**Review by doodlewind — APPROVED** [review]
Thanks for the feat!

**Pull request merged**